tests/test_sw_triangle.py

```diff
@@ -19,7 +19,7 @@ def setup_sw():
 
     fieldlist = ['u', 'D']
     timestepping = TimesteppingParameters()
-    output = OutputParameters(dumplist=(True,True), dirname='tests/sw')
+    output = OutputParameters(dumplist=(True,True), dirname=dirname)
```

Comment:
Oh yes, good point. It doesn't break because it doesn't run yet, it just sets up spaces and initial conditions.... but that should change soon!